docs/source/en/model_doc/fan.mdx

```diff
@@ -0,0 +1,78 @@
+<!--Copyright 2022 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# FAN
+
+## Overview
+
+The FAN model was proposed in [Understanding The Robustness in Vision Transformers](https://arxiv.org/pdf/2204.12451.pdf)  by Daquan Zhou, Zhiding Yu, Enze Xie,
```

Comment:
```suggestion
The FAN model was proposed in [Understanding The Robustness in Vision Transformers](https://arxiv.org/abs/2204.12451) by Daquan Zhou, Zhiding Yu, Enze Xie,
```
Please link to the abstract, rather than the PDF